unit_tests/utilities/test_zaza_utilities_upgrade_utils.py

```diff
@@ -102,6 +102,21 @@ def test_get_upgrade_groups(self):
             actual,
             expected)
 
+    def test_get_charm_upgrade_groups(self):
+        expected = [
+            ('Database Services', []),
+            ('Stateful Services', []),
+            ('Core Identity', []),
+            ('Control Plane', ['cinder']),
+            ('Data Plane', ['nova-compute']),
+            ('sweep_up', ['neutron-openvswitch', 'ntp'])]
+        actual = openstack_upgrade.get_charm_upgrade_groups()
+        pprint.pprint(expected)
+        pprint.pprint(actual)
```

Comment:
Is this debug left over from development?

Reply:
Thanks for the reviews! I'll clean those up.